micro_dl/cli/image_inference.py

```diff
@@ -169,7 +200,8 @@ def run_prediction(args, gpu_ids, gpu_mem_frac):
         predict=True,
     )
     print(model.summary())
-
+    model.compile(loss=loss_cls, optimizer='Adam', metrics=metrics_cls)
```

Comment:
I guess it doesn't matter much since we're not optimizing here, but you could get the optimizer from the config.